CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+# 0.8.0 (T.B.D)
+
+## Migration Disclaimer
+
+Version 0.8.0 of M3 switches M3DB to use file descriptors and the Read() system call for reading data files instead of mmaps.
+This should improve read latency and memory utilization for some workloads, as well as enables the ability to increase fetch concurrency past the default value.
```

Comment:
Can we link to anything detailing what types of workloads will be affected?

Reply:
Nothing to link to, but I can add a comment